### ISSUE TITLE
Create views in our DB to look like GERs

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#297]
_Imported from trac ticket 297.
Created by **jn@retailarchitects.com**
Opened in trac: Sep 23, 2010, 9:48:37 AM
Last modified in trac: Oct 11, 2010, 10:32:14 AM_
***
_by jn@retailarchitects.com:_
It would be advantageous to be able to use existing reports rather than have
to rewrite all the hundreds of reports before we can officially switch GERs
off. It seems that it would be possible to create a "GERs" view of our DB
that the reports can run against.There may be those that we will have to
write anyway, but it should speed up the day we turn GERs off.

It would also seem this would be a quick if dirty way to interface with
other systems like Route View, payroll, and any other systems.

**Comment by kentbower** [comment]
_**rneville** trac comment on Oct 11, 2010, 10:32:14 AM:_

Duplicate of #295

**Comment by kentbower** [comment]
_**rneville** trac status change on Oct 11, 2010, 10:32:14 AM:_ **closed**
